pallets/fixed-point/Cargo.toml

```diff
@@ -26,35 +42,28 @@ version = '1.3.0'
 
 [dependencies.frame-support]
```

Comment:
Probably a dumb question but why is there a preference for `[dependency.th-dependency]` syntax in the substrate ecosystem? In the wider rust community it is more common to write:

```
[dependencies]
frame-support = { version = "2.0.0-alpha.6", default-features = false }
…
…
```

Is there an advantage to use this more verbose formatting you think?

Reply:
I like the shorter syntax that you've shown as well, and I'm gradually updating the recipes to use it.

The reason so many Substrate-based projects use the long form, and the reason parts of the recipes use it, is because many of them are based off of the Substrate node template (https://github.com/substrate-developer-hub/substrate-node-template) which itself uses the long form. The template uses the long form because it is generated by [this script](https://github.com/paritytech/substrate/blob/master/.maintain/node-template-release.sh) which parses and re-writes the `Cargo.toml` file. In the transcription process, it outputs the long form.

So it isn't a conscious decision, just a side effect.